core/scss/core/colors/index.scss

```diff
@@ -0,0 +1,43 @@
+///
+// This file is for KSS generation only. To see colors check out:
+// scss/utilities/variables/core/_colors.scss
+///
+
+//
+// Colors
+//
+// Colors approved by Stanford University branding guidelines.
+//
+// $color-black -            #2e2d29; black
```

Comment:
This is alphabetical right now. Please feel the need to re-order if appropriate.